manifest_staging/charts/gatekeeper/templates/upgrade-crds-hook.yaml

```diff
@@ -93,10 +93,10 @@ spec:
       {{- end }}
       containers:
       - name: crds-upgrade
-        {{- if not .Values.image.release }}
-        image: '{{ .Values.image.crdRepository }}'
+        {{- if not .Values.preInstall.crdRepository.image.tag }}
+        image: '{{ .Values.preInstall.crdRepository.image.repository }}'
```

Comment:
if you are changing this, this needs to be defined in upgrade so crds can be updated

Reply:
Thanks for the hint. All tests are fixed now.